core/templates/pages/signup-page/signup-page.component.ts

```diff
@@ -169,8 +169,7 @@ angular.module('oppia').component('signupPage', {
           $window.location.href = decodeURIComponent(
             UrlService.getUrlParams().return_url);
         }, function(rejection) {
-          if (
-            rejection.data && rejection.data.status_code === 401) {
+          if (rejection.data && rejection.data.status_code === 401) {
```

Comment:
Weren't you planning to poll repeatedly in the case where the race condition works out badly?

Reply:
Ah, saw your comment above. Could you please file an issue that enumerates the next steps, so that the reviewer(s) can get the full picture of the chain of PRs and how this one fits into it?

Reply:
I have added steps to the linked issue.